src/molecules/modal/Modal.stories.js

```diff
@@ -20,6 +27,12 @@ storiesOf('Molecules/Modal', module)
     props: {
       contentKnob: {
         default: text('Content', 'Lorem ipsum dolor sit amet, consectetur adipLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.iscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.')
+      },
+      mainTransitionKnob: {
```

Comment:
I was telling you about adding these 2 knobs for transition but now I see that changing this type for the overlay is not that good, because only fade here makes sense. I think we can remove this one `mainTransitionKnob` and leave only `contentTransitionKnob` in all stories